### ISSUE TITLE
Announcement-Nachrichten

### ISSUE THREAD

**Issue by The-Compiler** [studentenportal/web#245]
Im Zuge von https://github.com/studentenportal/project/issues/6 hat jemand (@fabianhauser? @dbrgn?) ja diese Nachricht auf studentenportal.ch hinzugefügt:

> DRINGEND: Wir suchen eine/n neuen Studentenportal-Maintainer/in! Bitte unter github.com/studentenportal/project/issues/6 melden.

Das scheint aber nirgends im Git-Repo zu sein - ich nehme an, das wurde Live auf dem Server rein operiert? Wenn ich das richtig sehe, geht's um das hier in `apps/front/templates/front/base.html`:

```html
    {% block styles %}
        {% compress css %}
        <link rel="stylesheet" type="text/scss" href="{{ STATIC_URL }}sass/main.scss" />
        <link rel="stylesheet" href="{{ STATIC_URL }}css/responsive-nav.css" />
        {% endcompress %}
        <style>
        .maintainer {
          background-color: orange;
          border: 3px solid red;
          padding: 8px;
          margin-bottom: 16px;
          font-size: 1.3em;
          line-height: 1.3em;
        }
        </style>
    {% endblock %}
```

sowie

```html
    <div class="main">
        <div class="maintainer"><strong>DRINGEND:</strong> Wir suchen eine/n neuen Studentenportal-Maintainer/in! Bitte unter <a href="https://github.com/studentenportal/project/issues/6">github.com/studentenportal/project/issues/6</a> melden.</div>
```

Wir sollten vermutlich mal gucken, dass das noch im Repo landet - ist wohl nicht mehr soooo dringend, aber wär trotzdem schön, wenn sich jemand diesen Hut aufsetzen will. Ausserdem ists auch generell für Announcements praktisch.

**Comment by fabianhauser** [comment]
Ja, das hat @dbrgn vermutlich direkt dort abgelegt, super, dass du das gesehen hast! 👍 
Denke, wir können den Hinweis für den Moment weg lassen (hat eh nicht besonders viel genützt).

Aber evtl. wird das irgendwann wieder aktuell - würde dieses Issue daher vorübergehend offen lassen.

**Comment by dbrgn** [comment]
> Das scheint aber nirgends im Git-Repo zu sein - ich nehme an, das wurde Live auf dem Server rein operiert?

Ja das habe ich im Liveserver reingehackt :smile:
